version/1/4/hi/code_of_conduct.md

```diff
@@ -0,0 +1,74 @@
+आचार संहिता # योगदानकर्ता वाचा
+
+## हमारी प्रतिज्ञा
+
+एक खुला और स्वागत करते वातावरण को बढ़ावा देने के हित में, हम के रूप में
```

Comment:
स्वागत करते वातावरण is really a google translation. The real phrase for a "welcoming environment" would be something like "सुस्वागतमय वातावरण "